packages/test/test-gc-sweep-tests/src/fluidObjectTracker.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
```

Comment:
Do you want this one or common-utils one?

Reply:
Didn't realize there was a common-utils one, what's the difference, and I'm assuming it's preferred to use the common-utils one?

Reply:
I'd prefer the "assert" as we have more ability to input debug data. The other one feels more for customers. Since this is an internal test package, it will not affect bundle size.